### ISSUE TITLE
Invalid licenseUrl in nuspec

### ISSUE THREAD

**Issue by Greg-Smulko** [commandlineparser/commandline#429]
The `licenseUrl` in the `commandlineparser.nuspec` file is invalid. 

The nuspec file for ver. 2.4.3 contains:
```
<licenseUrl>https://raw.githubusercontent.com/gsscoder/commandline/master/doc/LICENSE</licenseUrl>
<projectUrl>https://github.com/gsscoder/commandline</projectUrl>
<iconUrl>https://raw.githubusercontent.com/commandlineparser/commandline/master/art/CommandLine20.png</iconUrl>
```

The  `https://raw.githubusercontent.com/gsscoder/commandline/master/doc/LICENSE` doesn't exist. It should be `https://raw.githubusercontent.com/commandlineparser/commandline/master/License.md` instead.
The `projectUrl` element could be update too.

I'd open a PR for this, but can't find the nuspec file in this repo. :(

**Comment by Tyrrrz** [comment]
There's no nuspec, you have to update csproj:
https://github.com/commandlineparser/commandline/blob/030eb8d431c0c56b62277c654d0951308e873960/src/CommandLine/CommandLine.csproj#L21

**Comment by Greg-Smulko** [comment]
Fixed already in #549, as reported in #545, so closing it.
